### ISSUE TITLE
Spurious values reported for battery charge/discharge energy totals

### ISSUE THREAD

**Issue by jamesremuscat** [britkat1980/giv_tcp#265]
For some reason, GivTCP reported a sudden jump in `Inverter energy Battery Charge Energy Total kWh` this morning:

![image](https://github.com/user-attachments/assets/9689815d-4aa2-4abe-a79b-cad9b678ef31)

And another spurious jump in `Inverter energy Battery Discharge Energy Total kWh` later on:

![image](https://github.com/user-attachments/assets/e068897e-6b7b-44a4-9354-c4ce4cfb7f64)

I can't find an analagous value through the GivEnergy portal, but the "battery throughput total" values look normal.

There are no entries in the Docker container log for those time periods.

Unlike the random spikes discussed elsewhere (#252, https://github.com/britkat1980/ha-addons/issues/20) these erroneous values persisted until I restarted the container (the steep drop to the right of both charts).

The equvalent `Today kWh` values have been unreliable due to the data spikes issue:
![image](https://github.com/user-attachments/assets/f3fff492-9b26-4a44-8ad7-d8b3b9f875e7)
Those spikes don't seem to correspond to the jumps in the total values.

EDIT: Should have specified, I'm running 3.0.3.

**Comment by shuntfield** [comment]
I am also getting the step change, it adds around 2200+ kWh (2 megawatts!) around about 4pm every day. It might be in coincidence with the time when I first started the inverter, it was installed two days ago, and two days around 4pm its done the same. 

I have too, now restarted the container and its come back to normal.
![image](https://github.com/user-attachments/assets/4c51df69-fd33-40ba-8a62-e658d9462c10)

I have deleted the error value from the stats and restarted the container, but the number comes from nowhere as you suggest @jamesremuscat

**Comment by shuntfield** [comment]
I am also getting the step change, it adds around 2200+ kWh (2 megawatts!) around about 4pm every day. It might be in coincidence with the time when I first started the inverter, it was installed two days ago, and two days around 4pm its done the same.

I have too, now restarted the container and its come back to normal.
![image](https://github.com/user-attachments/assets/2d3cc006-12c2-41b3-b2be-0a036fe8d965)

I have deleted the error value from the stats and restarted the container, but the number comes from nowhere as you suggest @jamesremuscat
